src/app/components/text-with-link/text-with-link.tsx

```diff
@@ -0,0 +1,19 @@
+import { Fragment } from 'react/jsx-runtime';
+import { LinkList } from '@/features/required/required.types';
+
+import './text-with-link.scss';
+
+interface TextWithLinkProps {
+  data: LinkList;
+}
+
+export const TextWithLink = ({ data }: TextWithLinkProps) => {
+  return data.map((item, index) => (
+    <Fragment key={index}>
+      {item.text && <span>{item.text}</span>}
```

Comment:
Avoid using array index as a key when mapping elements in React. This can potentially lead to inconsistencies with your UI. Consider using a unique value, like an item ID, if available.

Reply:
Done